src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int randomColor = random.nextInt(Colors.values().length);
+        return Colors.values()[randomColor].toString();
```

Comment:
Instead of using `toString()`, you should use the `name()` method to get the String representation of enum constants. This ensures that even if the `toString()` method is overridden in the enum, you will get the correct name of the enum constant.